### ISSUE TITLE
Structures that use a base structure can't be imported again after exporting the header from binja

### ISSUE THREAD

**Issue by 0x61616161** [Vector35/binaryninja-api#4220]
**Version and Platform (required):**
- Binary Ninja Version: 3.4.4225-dev 
- OS: Windows
- OS Version: 10
- CPU Architecture: x64

**Bug Description:**
After adding structures with the new base structure feature and exporting the header from binja, it can not be imported again. (used an .exe file for this, but seems unrelated to the actual file analyzed)

**Steps To Reproduce:**
Please provide all steps required to reproduce the behavior:
1. Create two new structures:

```c 
struct struct_1 __packed
{
    int32_t a;
};

struct __base(struct_1, 0) struct_2 __packed
{
    __inherited int32_t struct_1::a;
};
```

 or use gui: 
![struct](https://user-images.githubusercontent.com/93497647/232143030-be8ac2c2-bb29-40b3-836b-a1f821614fad.jpg)

2. Click on Analysis, Export header file
![export](https://user-images.githubusercontent.com/93497647/232143054-0259f788-754f-460d-8446-717cd7928c35.jpg)
3. Open a new binary
4. Click on Analysis, Import header file and select the previously exported file
5. Click import and see

![import](https://user-images.githubusercontent.com/93497647/232142968-e6af913b-71f5-4d62-95ea-80fb16384ae0.jpg)

**Expected Behavior:**
It is possible to import the previously created structures.

**Additional Information:**
Attached the generated .h file (renamed as .txt for upload)
[resources2.txt](https://github.com/Vector35/binaryninja-api/files/11236610/resources2.txt)

**Comment by CouleeApps** [comment]
Looks like TypePrinter needs to be updated to support the new inheritance attributes and syntax

**Comment by CouleeApps** [comment]
Well in the process of trying to fix this i've managed to find 2 additional bugs:
https://github.com/Vector35/binaryninja-api/issues/4222
https://github.com/Vector35/binaryninja-api/issues/4221

**Comment by CouleeApps** [comment]
This will be fixed in 3.4.4227-dev.
